base/regex.jl

```diff
@@ -559,7 +571,7 @@ end
 *(r::Regex) = r # avoids wrapping r in a useless subpattern
 
 unwrap_string(r::Regex, unshared::UInt32) = string("(?", regex_opts_str(r.compile_options & unshared), ':', r.pattern, ')')
-unwrap_string(s::Union{AbstractString,AbstractChar}, ::UInt32) = s # no need to wrap in subpattern
+unwrap_string(s::Union{AbstractString,AbstractChar}, ::UInt32) = string("\\Q", s, "\\E")
```

Comment:
Ok, you're going to want to kill me but what if `s` contains `\E`?

Reply:
Hahaha! actually, I should have known better with my background in cybersecurity (cf. injection attacks) ;-)